02/compose.yml

```
@@ -0,0 +1,28 @@
+services:
+  praha-challenge-db:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: praha-challenge-db-02
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
+      - MYSQL_DATABASE=employees
+    ports:
+      - 3306:3306
+    volumes:
+      - ./initdb.d:/docker-entrypoint-initdb.d
+      - praha-challenge-db-data-02:/var/lib/mysql
```

- `Dockerfile`内で`/docker-entrypoint-initdb.d`にいろいろ格納
- さらにvolume mout
- 【`Dockerfile`内で`/docker-entrypoint-initdb.d`にいろいろ格納】したやつ全部消える（上書きされるから）

...なので、volume mountしちゃだめです！
```suggestion
    volumes:
      - praha-challenge-db-data-02:/var/lib/mysql
```